content/en/synthetics/multistep.md

```diff
@@ -27,13 +27,13 @@ further_reading:
 
 Multistep API tests allow you to chain several [HTTP requests][1] at once to proactively monitor and ensure that the sophisticated journeys on your key services are available at anytime, and from anywhere. If you want to perform single requests to your services, leverage [API tests][2].
 
-You can do the following actions:
+You can accomplish the following:
 
 * Execute HTTP requests on API endpoints requiring authentication (for example, through a token)
 * Monitor key business transactions at the API level
 * Simulate end-to-end mobile application journeys
 
-{{< img src="synthetics/multistep_tests/multistep_test_steps.png" alt="Multiple test requests as steps in a multistep API test" style="width:90%;" >}}
+{{< img src="synthetics/api_tests/ms_overview.png" alt="Multistep API test overview" style="width:90%;" >}}
```

Comment:
This image looks blurry in the preview. I'm not sure if it's a new one or an old one, but it's probably worth retaking it.